include/bluetooth/mesh/scene_srv.h

```diff
@@ -56,6 +57,7 @@ struct bt_mesh_scene_srv;
  *  @param[in] _srv Pointer to a @ref bt_mesh_scene_srv instance.
  */
 #define BT_MESH_MODEL_SCENE_SRV(_srv)                                          \
+	BT_MESH_MODEL_DTT_SRV(&(_srv)->dtt),                               \
```

Comment:
We deliberately left this out in the original code, as this will make it impossible to put a scene server and a ponoff server on the same element (and by extension, any of the lightness models). Additionally, the extension API only supports a single parent, so the DTT server can't be extended by both the ponoff server and the scene setup server. 

We would first need to implement a new version of the extension data structure to make this work. IMO, we still shouldn't add the DTT server to the composition data in this macro, as it's incompatible with other models. Arguably, it might make more sense to remove the DTT server from these macro in the ponoff server as well, and force users to explicitly add DTT to the composition data.

Reply:
Note that the Scheduler setup server (and thus scene server) per Mdlv1.01 may only be on primary element. They are thus leveraging existing ponoff/dtt servers.

Reply:
@eanderlind What do you mean by laveraging existing ponoff/dtt servers?

Reply:
Scheduler setup server table 5.49 doesn’t show any onoff server
which would have been there if ponoff srv was extended fully. 
Our interpretation is that the scheduler and schedule register
setup servers (per spec 5.4.3.1 always on primary element) should use dtt/ponoff srv
discovered per rules in dtt srv section and not create a strict extension 
tree as is typical.